docker-compose.yml

```diff
@@ -139,7 +139,7 @@ services:
     restart: always
     logging: *default-logging
   search-query-management:
-    image: lblod/toezicht-search-query-management-service:0.3.6
+    image: lblod/toezicht-search-query-management-service:0.3.7-rc.1
```

Comment:
This image doesn't seem to exist. I think we need a Woodpecker setup in that repo for it to create images now.

Also, instead of RC tagged releases it would be better to just use the ["feature" image](https://github.com/redpencilio/ci-templates/blob/master/woodpecker/service/.feature.yml) instead.

Reply:
I've replaced this with the feature image, so it also works even if you don't have the image locally in your overrides. We can swap it with the real version once we make a new release of the service.